manual/src/threelang/tfchain/tfchain.md

```diff
@@ -0,0 +1,237 @@
+# TF Chain Actions
+In a simple format you can do almost all the extrinsic and queries on the chian. List of the namespaces
+
+## Name spaces and their actions
+- [Account](#account)
+- [Balance](#balance)
+- [Twins](#twins)
+- [Nodes](#nodes)
+- [Farms](#farms)
+- [Contracts](#contracts)
+- [Service Contract](#service-contract)
+- [Metadata](#metadata)
+
+
+## Account
+- Create new account on the chain by Generating mnemonic
+    ```md
+    !!tfchain.account.create
+        network:devnet 
+    ```
+    - `network`: is the tfchain network, should be one of (mainnet, testnet, qanet, devnet)
+- Load the account to do chain extrinsic 
+    ```md
+    !!tfchain.client.load
+        network:devnet
+        mnemonic:\'secret words as your mnemonic\'
+    ```
+    - `network`: is the tfchain network, should be one of (mainnet, testnet, qanet, devnet)
+    - `mnemonic`: twin mnemonic on the chain
+
+## Balance
+- Transfer balance between tfchain twins
+    ```md
+    !!tfchain.balance.transfer
+        amount:100
+        destination:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY
+    ```
+    - `amount`: the amount of sent TFTs
+    - `destination`: tfchain address in SS58 format
+- Swap balance to stellar account
+    ```md
+    !!tfchain.balance.swap
+        amount:100
+        destination:GCCVPYFOHY7ZB7557JKENAX62LUAPLMGIWNZJAFV2MITK6T32V37KEJU
+    ```
+    - `amount`: the amount of sent TFTs
+    - `destination`: stellar address
+- Get current balance for tfchain account
+    ```md
+    !!tfchain.balance.get
+        address:5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY   
```

Comment:
The address can be optional. In case the user doesn't provide it it will return the balance of the loaded key